particula/particles/properties/sort_bins.py

```diff
@@ -0,0 +1,45 @@
+"""
+Sort bins of distribution representation
+"""
+from typing import Union
+import numpy as np
+from numpy.typing import NDArray
+
+
+def get_sorted_bins_by_radius(
+    radius: NDArray[np.float64],
+    distribution: NDArray[np.float64],
+    concentration: NDArray[np.float64],
+    charge: Union[NDArray[np.float64], float],
+) -> tuple[
+    NDArray[np.float64],
+    NDArray[np.float64],
+    Union[NDArray[np.float64], float],
+]:
+    """Ensure distribution bins are sorted by increasing radius.
+
+    Arguments:
+        - radius : The radii of the particles.
+        - distribution : The distribution of particle sizes or masses.
+        - concentration : The concentration of each particle size or mass.
+        - charge : The (optional) charge per particle; can be a scalar or NumPy array.
+
+    Returns:
+        - distribution : The sorted distribution of particle sizes or masses.
+        - concentration : The sorted concentration of each particle size/mass.
+        - charge : The sorted charge of each particle size/mass.
+    """
+    sort_index = np.argsort(radius)
+    sorting_is_needed = not np.array_equal(sort_index, np.arange(radius.size))
+
```

Comment:
Do you have opinions on explicit if statement variable names? vs putting the logic right in the if statement. 

I saw Uncle Bob do this, I think it is clearer to read, but then you get these additional statements before each if. I think for complex if statements it is good, but for simple ones don't do it?